r/src/symbols.cpp

```diff
@@ -25,6 +25,16 @@ SEXP symbols::dot_Internal = Rf_install(".Internal");
 SEXP symbols::inspect = Rf_install("inspect");
 SEXP symbols::row_names = Rf_install("row.names");
 
+SEXP get_classes_POSIXct() {
```

Comment:
What exactly is this for? And should it be used in `array_to_vector.cpp` too, where we make `POSIXct` vectors?

Reply:
it just is so that we'd only build that character vector once. It should indeed also be used in `array_to_vector.cpp` too.

Reply:
Done